buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -120,6 +120,7 @@ class BuildPlugin implements Plugin<Project> {
                 println "  JDK Version           : ${gradleJavaVersionDetails}"
                 println "  JAVA_HOME             : ${gradleJavaHome}"
             }
+            println " Random Testing Seed    : ${project.testSeed}"
```

Comment:
In fact, I think it will be nice because I can then work on adding a `beast` task like we have in lucene which will generate separate test tasks per "iteration" of the beast run, with unique seeds.